src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -1,180 +1,165 @@
     />
   );
 
+  const navDropdownClassnames = classNames("NavSearchBar__nav-dropdown", {
+    "NavSearchBar__nav-dropdown--show": searchQuery,
+  });
+
   return (
-    <div className="nav-search-links">
+    <div className="NavSearchBar">
       <InputField
-        className="nav__search"
-        value={searchQuery}
+        containerClassName="NavSearchBar__search-input-container"
+        inputClassName="NavSearchBar__search-input"
         onChange={onSearchInputChange}
         placeholder="Search for people, rooms, events..."
         autoComplete="off"
         iconStart={faSearch}
         iconEnd={isTruthy(searchQuery) ? clearSearchIcon : undefined}
       />
 
-      {numberOfSearchResults > 0 && (
-        <div className="nav-search-results">
-          <div className="nav-search-result-number">
-            <b>{numberOfSearchResults}</b> search results
-          </div>
-
-          {/* @debt we really shouldn't be using the index as part of the key here, it's unstable.. but rooms don't have a unique identifier */}
-          {searchResult.rooms.map((room, index) => {
-            return (
-              <div
-                className="row"
-                key={`room-${room.title}-${index}`}
-                onClick={() => {
-                  setSelectedRoom(room);
-                  clearSearchQuery();
-                }}
-              >
-                <div
-                  className="result-avatar"
-                  style={{
-                    backgroundImage: `url(${room.image_url})`,
-                  }}
-                />
-                <div className="result-info">
-                  <div className="result-title">{room.title}</div>
-                  <div>Room</div>
-                </div>
-              </div>
-            );
-          })}
-
-          {searchResult.events.map((event) => {
-            return (
-              <div className="row" key={`event-${event.id ?? event.name}`}>
-                <div>
-                  <div>{event.name}</div>
-                  <div>Event</div>
-                </div>
-              </div>
-            );
-          })}
-
-          {searchResult.users.map((user) => {
-            return (
-              <div
-                className="row"
-                key={`user-${user.id}`}
-                onClick={() => {
-                  openUserProfileModal(user);
-                  clearSearchQuery();
-                }}
-              >
-                <div
-                  className="result-avatar"
-                  style={{
-                    backgroundImage: `url(${user.pictureUrl})`,
-                  }}
-                />
-                <div className="result-info">
-                  <div>{user.partyName}</div>
-                </div>
-              </div>
-            );
-          })}
+      <div className={navDropdownClassnames}>
+        <div className="NavSearchBar__nav-dropdown__title mb--05 mt--05 font-size--small">
```

Comment:
[NB] I'm still not convinced that using `mb--05` and similar classes directly in our JSX is nicer/better than having the 'defining class' (eg. `NavSearchBar__nav-dropdown__title`) containing all of the relevant styles. It kind of feels like we are pushing the 'styling layer' into the 'structure layer'.

If we want it for ease of use purposes, then we could make them SCSS mixins/similar?

@mike-lvov @denisdimitrov What do you think? Any opinions one way or another?

Reply:
removed those class names: https://github.com/sparkletown/sparkle/pull/1328/commits/5c73125a614d77c2669161c358485df8ad2c7e9d